src/Scan/RestScans.ts

```diff
@@ -89,6 +89,16 @@ export class RestScans implements Scans {
     await this.client.delete(`/api/v1/scans/${scanId}`);
   }
 
+  public async isProjectExist(projectId: string): Promise<boolean> {
+    try {
+      await this.client.get(`/api/v1/projects/${projectId}`);
+    } catch (error) {
+      return false;
```

Comment:
for future PRs: handling errors in such way will hide authentication/authorization errors as there could be 401 or 403 error but user will get "The specified project ID ${projectId} does not exist" when in reality project exist and there problem with token actually